ledger/participant-integration-api/src/test/suite/scala/platform/store/appendonlydao/events/ACSReaderSpec.scala

```diff
@@ -224,60 +224,56 @@ class ACSReaderSpec extends AsyncFlatSpec with Matchers with BeforeAndAfterAll {
       _ <- puppetTask1.started
       _ <- puppetTask2.started
       _ <- puppetTask3.started
-      puppetTask10 = {
+      (puppetTask10, puppetTask5) = {
```

Comment:
This test was testing full detailed sequencing with mapAsyncUnordered, as which have changed here, the test needed substantial adaptation.